test/testlib/nuodb_database_utilities.go

```diff
@@ -190,9 +199,13 @@ func StartDatabaseTemplate(t *testing.T, namespaceName string, adminPod string,
 		if opt.DbPrimaryRelease {
 			smPodName0 := GetPodName(t, namespaceName, smPodNameTemplate)
 			AddTeardown(TEARDOWN_DATABASE, func() {
-				for _, smPod := range GetPodNames(t, namespaceName, smPodNameTemplate) {
-					t.Logf("Getting log from SM pod: %s", smPod)
-					go GetAppLog(t, namespaceName, smPod, "", &v12.PodLogOptions{Follow: true})
+				pods, err := findPods(t, namespaceName, smPodNameTemplate)
+				if err != nil {
```

Comment:
It looks like `findPods()` just returns non-`nil` error when the list of pods is empty. Wouldn't the subsequent loop just do nothing in that case?

If there are any pods that are somehow returned by `findPods()`, we would want to collect their logging regardless of the error, right?

Reply:
I haven't checked the `findPods` implementation in detail but used the usual _go_ error handling which guards against returned `nil` slice. As you pointed out, we could omit the error handling in this specific case.